great-idea/css/index.css

```diff
@@ -63,4 +63,145 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+	.navigation{
+		display: flex;
+		justify-content: space-around;
+		align-items: center;
+	
+	}
+	.navigation a {
+		padding: 30px;
+		border: 0;
+		font-size: 100%;
+		color: grey;
+		text-decoration: none;
+		list-style-type: none;
+	}
+	.container{
```

Comment:
Adding a width to your container would make the page look more like the design file. For this project, 876px is the width of the content